xpod.go

```diff
@@ -8,99 +9,146 @@ import (
 	"log"
 	"net/http"
 	"os"
+	"time"
 )
 
 var t_site = "https://xray.fm"
 var t_shows = t_site + "/shows"
 
-func fetch(url string) (*goquery.Document, error) {
+// fetch url, returning either a parsed goquery Document or an error.
+func fetch(url string) (*http.Response, *goquery.Document, error) {
 	var doc *goquery.Document
 
-	res, err := http.Get(url)
+	resp, err := http.Get(url)
 	if err != nil {
-		return doc, errors.Wrapf(err, "Failed to fetch: `%s'", url)
+		return nil, nil, errors.Wrapf(err, "Failed to fetch: `%s'", url)
```

Comment:
```suggestion
		return nil, nil, errors.Wrapf(err, "Failed to fetch: %q", url)
```

`%q` handles quoting things for you.